vuepress/docs/next/tutorials/devops/plugin-environment-variables.md

```diff
@@ -2,60 +2,54 @@
 sidebarDepth: 2
 ---
 
-# Use Plugin Environment Variables to Customize Microservices
+# Use Environment Variables to Customize Microservices
 
-This tutorial describes how to use environment variables to customize the behavior of plugin microservices. Using environment variables as a configuration mechanism allows the same Linux image to run in development and production environments. This is especially useful when managing sensitive information via Kubernetes Secrets.
+This tutorial describes how to use environment variables to customize the behavior of microservices. Using environment variables as a configuration mechanism allows the same Linux image to run in development and production environments. This is especially useful when managing sensitive information via Kubernetes Secrets.
 
-Version 4 of the Entando plugin descriptor adds the ability to make environment variables available to a microservice via the plugin `descriptor.yaml` file. There are two options.
+The `entando.json` file makes environment variables available to a microservice in a docker-based bundle. There are two options:
 1. Inject the variables using a key-value pair
 2. Inject a reference to an existing Kubernetes Secret 
 
-This tutorial will demonstrate both of these options. The Entando Plugin will receive one environment value directly as plain text in the Pod YAML and the other indirectly via a referenced Secret.
+This tutorial will demonstrate both of these options. The microservice will receive one environment value directly as plain text in the pod YAML and the other indirectly through a referenced Secret.
 
 ## Prerequisites
-* [A project bundle](../create/ms/generate-microservices-and-micro-frontends.md) using the latest Entando Blueprint. 
+* [A bundle project using the latest Entando Blueprint](../create/ms/generate-microservices-and-micro-frontends.md) 
 
-## Add Environment Variables to the Plugin Descriptor File
-1. Determine YOUR-BUNDLE-ID. You can do this from your project directory using this command:
-```sh
-ent prj get-bundle-id
-```
-Alternatively, supply the full bundle URL (including the .git suffix):
+## Add Environment Variables to the Microservice
+1. To determine YOUR-BUNDLE-ID, run the following command from the root project directory. Supply the full bundle URL (including the .git suffix).
 ```sh
 ent ecr get-bundle-id https://github.com/YOUR-ACCOUNT/YOUR-PLUGIN-BUNDLE.git
```

Comment:
maybe I'm confused, should this be docker? 

And it was not a good idea originally to call this a PLUGIN-Bundle, should change the name

Reply:
good catch - copy/paste error

agree